### ISSUE TITLE
Pressing ENTER starts music player without warning

### ISSUE THREAD

**Issue by salty-horse** [ValveSoftware/steam-for-linux#3516]
Using the Sep 22 2014, 20:22:29 build.
1. Have some soundtracks available through Steam.
2. Go to the Steam Library using the "Game Details" view.
3. Choose a game and start it. Quit the game.
4. Press ENTER on the keyboard.

Result:
The Steam music player shows up and automatically starts playing music - in my case, it plays the Gone Home soundtrack.

Expected result:
The game should start again. The music player should not start.

**Comment by rpatterson** [comment]
This happens on Windows too.

**Comment by rlpowell** [comment]
Drives me so crazy.  ;_;  Windows.

**Comment by sgtnoodle** [comment]
I just opened up steam in Arch Linux. A sound player seemingly randomly popped up and I immediately hit the x in the corner. Steam proceeded to play some obnoxious payday 2 themed 12 days of Christmas parody. I closed the steam client but the sound remained, so I typed "killall steam" at the command line and the music stopped.

**Comment by ysu2** [comment]
This happens to me lately a lot.  It seems still unfixed.

**Comment by DeHackEd** [comment]
I've had this happen lately on Linux as well. I'm not aware of any UI interaction on Steam itself that would cause this as I do so fairly little. Sometimes pressing Enter will launch the highlighted game, sometimes (rarely but it happens) the music player starts up.

**Comment by slottybon** [comment]
Yup. Happens to me every time my machine takes a nap with steam running in the background. 
Upon login, when I enter my password and hit the ENTER key it logs me in an starts the steam player.

Windows 10 here

**Comment by jmenashe** [comment]
Happening to me now on Windows 10.

**Comment by zatricky** [comment]
I just had this happen to me. :-|

I launched Steam then changed focus to my browser. I pressed Enter as Steam stole focus. Suddenly blaring music from Crimsonland. Clicked the [x] thinking that would stop the music - but no. It then took me a few minutes to figure out how to stop it. :-/

**Comment by DeHackEd** [comment]
Can confirm, it just happened to me yesterday for the first time in quite a while.

**Comment by Wilbert78** [comment]
Happening to me with the soundtrack of Baldur's gate 2. Really irritating.

**Comment by KingLem** [comment]
Still happening to me, and closing the window doesn't stop the music.

**Comment by ReactionAndy** [comment]
Just had it happend for me on Windows after tabbing out of a game to check on the download progress of another game and hitting Enter

**Comment by c67f** [comment]
Just happened to me for XCOM Enemy Unknown/Within. I didn't even know steam had a sound/music player!

**Comment by SebOuellette** [comment]
Just happened to me... pressed enter and elevator music started playing. What tf is this bug?

**Comment by Dougomite** [comment]
Ha yeah same, I had a bunch of Steam windows open. I think I had just pasted something in the search box, hit enter and got the music player playing elevator music. Closed the player...but then had to figure out how to get it back because the music of course didn't stop.

Is this just some A/B test to see if people would use the music player if they knew it existed?
